src/components/Nav/NavLink.jsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 import { LinkButton } from './styles';
 
 const NavLink = ({
   title, value, index, onClickLink,
 }) => (
-  <LinkButton
-    onClick={() => onClickLink(index)}
-    value={value}
-    index={index}
-  >
-    {title}
-  </LinkButton>
+  <Link to="/movies">
```

Comment:
This link should not redirect to ```/movies```?
(Because if we are on the Description page and want to open Popular movies, it should redirect to it)